onboarding/include/amb_light_service.h

```diff
@@ -15,7 +15,8 @@ typedef enum {
 
 /* USER CODE BEGIN */
 // Define light service queue config here
-
+#define QUEUE_SIZE 2
```

Comment:
A size of 2 should be fine because we're only sending 1 event to the queue every second. As a result, it's unlikely that the queue will be full. However, this does depend on a few things in your implementation. 

For example, let's say we try `xQueueReceive` with a block time of 10 seconds. Then, if there's nothing in the queue, the light service task will enter the blocked state for 10 seconds. In those 10 seconds, the controller task will try and send an event at least 9 times. After 2, the queue will be full. Although, this is an exaggeration since your block time will likely be set to <1 second.

Just to be safe, I'd make the size bigger (like 10).